app/data/management/commands/remove_old_exports.py

```diff
@@ -0,0 +1,23 @@
+import os
+import re
+import time
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Delete old CSV export files'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--max-age', type=int, default=6,
+                            help="Oldest files to keep, in hours (default: 6).")
+
+    def handle(self, *args, **options):
+        path = settings.CELERY_EXPORTS_FILE_PATH
+        min_mtime = time.time() - (options['max_age'] * 3600)
+        for filename in os.listdir(path):
+            file = os.path.join(path, filename)
```

Comment:
`file` is a Python built-in so it might be better to rename this variable although I think this will still work.